tests/validation/cattlevalidationtest/core/test_lb.py

```diff
@@ -971,6 +971,8 @@ def test_lb_with_health_check_without_uri(client):
     delete_all(client, [lb, con1])
 
 
+@pytest.mark.skipif(
+    True, reason='round robin access check does not always apply')
```

Comment:
yes. I am not able to reliably test for round robin access for LBs with "leastconn" . It sometimes does not not do round robin.